cirq-core/cirq/transformers/routing/route_circuit_cqc_test.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+import pytest
+
```

Comment:
Can you add tests for circuits containing:
1) Measurements + Classically controlled operations
2) Multi qubit measurements. 
3) 2-qubit non-unitary gates like `DepolarizingChannel` on 2-qubits.
4) `cirq.GlobalPhaseGate`s : test that these are not dropped after routing (they act on 0 qubits)
5) Operations tagged with custom user tags: The user tags should be preserved as part of routing.  

These can be small circuits just to validate that qubit routing is working as expected.